### PR TITLE
Support Ladybug database storage

### PR DESCRIPTION
**Pull request by jacodg**

### CODE REVIEW

File: ladybug/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	<name>Frank!Framework Ladybug</name>
 
 	<properties>
-		<ladybug.version>2.3-20230911.183804</ladybug.version>
+		<ladybug.version>2.3-20240103.172229</ladybug.version>
```
Comment (jkosternl) [inline ladybug/pom.xml:14]: Does this version bump also fixes vulnerabilities of the current (older) Ladybug version?

**Review by jkosternl — APPROVED** [review]
Looks fine. Where can I find exactly what is stored in the database by Ladybug or how to use it Jaco? I just miss some background knowledge, but perhaps that's already documented somewhere?

File: ladybug/pom.xml (hunk 1 above)
Comment (jacodg) [reply ladybug/pom.xml:14]: Yes, it is the latest version which includes:

https://github.com/wearefrank/ladybug/blob/master/RELEASES.md

We still need to improve/automate the release mechanism for Ladybug.

**Comment by jacodg** [comment]
See the database changelog files at:

https://github.com/wearefrank/ladybug/tree/master/src/main/resources/ladybug

Basically the Frank!Developer doesn't have to do anything. When jdbc.migrator.active is true (which is generally already used by Franks that have the rights to make database changes) Ladybug will create it's tables and use the database storage classes. If not, it will fall back to using the file storage classes storing data in log.dir (as it currently does).

File: ladybug/pom.xml (hunk 1 above)
Comment (jkosternl) [reply ladybug/pom.xml:14]: Ok great!

File: ladybug/src/main/java/org/frankframework/ibistesttool/DeploymentSpecificsBeanPostProcessor.java [hunk 2]
```diff
@@ -92,6 +93,14 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
 				}
 			}
 		}
+		if (bean instanceof SpringLiquibase) {
+			boolean active = APP_CONSTANTS.getBoolean("ladybug.jdbc.migrator.active",
+					APP_CONSTANTS.getBoolean("jdbc.migrator.active", false));
+			if (!active) {
```
Comment (tnleeuw) [inline ladybug/src/main/java/org/frankframework/ibistesttool/DeploymentSpecificsBeanPostProcessor.java:99]: Waarom niet gewoon altijd de waarde van `active` in de bean zetten?

Comment (nielsm5) [reply ladybug/src/main/java/org/frankframework/ibistesttool/DeploymentSpecificsBeanPostProcessor.java:99]: Als private field (bovenaan in de classe)

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**